src/sections/Partners/index.js

```diff
@@ -1,243 +1,196 @@
-              <a href="https://www.microsoft.com/" target="_blank" rel="noopener noreferrer">
-                <img width="120%" height="75px" src={partner8} title="Click to know More about our partner" alt="Microsoft" />
-              </a>
-            </div>
-          </Col>
-          <Col xs={12} sm={10} lg={10}>
-            <div rightAlign={true} className="container">
-              <h2>Microsoft</h2>
-              <div>
-                Microsoft Azure is an ever-expanding set of cloud services to help your organization meet your business challenges. It’s the freedom to build, manage, and deploy applications on a massive, global network using your favorite tools and frameworks.
-              </div>
-            </div>
-          </Col>
-        </Row>
-        <Row>
-          <Col xs={12} sm={10} lg={10}>
-            <div leftAlign={true} className="container">
-              <h2>Red Hat</h2>
-              <div>
-                The world’s leading provider of enterprise open source solutions, using a community-powered approach to deliver high-performing Linux, cloud, container, and Kubernetes technologies.
-              </div>
-            </div>
-          </Col>
-          <Col xs={12} sm={2} lg={2}>
-            <div className="img1" style={{ paddingTop: "1.625em" }}>
-              <a href="https://www.redhat.com/" target="_blank" rel="noopener noreferrer">
-                <img width="130px" height="100%" src={partner9} title="Click to know More about our partner" alt="Red Hat" />
-              </a>
-            </div>
-          </Col>
-        </Row>
-        <Row>
-          <Col xs={12} sm={2} lg={2}>
-            <div className="img1" style={{ paddingTop: "2em" }}>
-              <a href="https://smi-spec.io/" target="_blank" rel="noopener noreferrer">
-                <img width="100%" height="60px" src={partner10} title="Click to know More about our partner" alt="Service Mesh Interface" />
-              </a>
-            </div>
-          </Col>
-          <Col xs={12} sm={10} lg={10}>
-            <div rightAlign={true} className="container">
-              <h2>Service Mesh Interface</h2>
-              <div>
-                The Service Mesh Interface (SMI) is a specification for service meshes that run on Kubernetes. It defines a common standard that can be implemented by a variety of providers. This allows for both standardization for end-users and innovation by providers of Service Mesh Technology. It enables flexibility and interoperability.
-              </div>
-            </div>
-          </Col>
-        </Row>
-        <Row className='partner-button-row'>
-          <Col xs={12} sm={4} lg={4}>
-            <div className="container partner-button">
-              <Button primary title="BECOME A PARTNER" url="mailto:partners@layer5.io" external="true" />
-            </div>
-          </Col>
-        </Row>
+        {
+          technology_partners.map((partner,index) => (
```

Comment:
Is it possible to reuse the merge the code into one?

Reply:
Yes It can be merged .I'll merge it .